net-im/psi/psi-9999.ebuild

```diff
@@ -66,16 +73,9 @@ pkg_setup() {
 	if use extras; then
 		MY_PN=psi-plus
 		echo
-		ewarn "You're about to build heavily patched version of Psi called Psi+."
-		ewarn "It has really nice features but still is under heavy development."
-		ewarn "Take a look at homepage for more info: https://code.google.com/p/psi-dev"
-		ewarn "If you wish to disable some patches just put"
-		ewarn "MY_EPATCH_EXCLUDE=\"list of patches\""
-		ewarn "into /etc/portage/env/${CATEGORY}/${PN} file."
-		echo
-		ewarn "Note: some patches depend on other. So if you disabled some patch"
-		ewarn "and other started to fail to apply, you'll have to disable patches"
-		ewarn "that fail too."
+		ewarn "You're about to build patched version of Psi called Psi+."
+		ewarn "It has new nice features not yet included to Psi."
+		ewarn "Take a look at homepage for more info: http://psi-plus.com/"
```

Comment:
That's Psi+ home. HOMEPAGE is just Psi. But thanks. maybe I will use it somewhere else :-)